Modules/User/Http/Controllers/RolesController.php

```diff
@@ -25,4 +38,18 @@ public function getAllRoles()
     {
         return Role::all();
     }
+
+    public function addRole(RoleRequest $request)
+    {
+        $this->authorize('viewAny', User::class);
+        $validated = $request->validated();
+        $this->service->addRole($validated);
+
+        return redirect()->back();
+    }
+
+    public function deleteRole(Role $role)
+    {
+        Role::find($role->delete());
```

Comment:
How do you this is working? Anyone can say that it is incorrect just by seeing it
```suggestion
        $role->delete();
```